src-java/kilda-model/src/main/java/org/openkilda/model/FlowPath.java

```diff
@@ -115,14 +116,16 @@
     @Setter(AccessLevel.NONE)
     private List<PathSegment> segments = new ArrayList<>();
 
-
+    @Property(name = "enabled_applications")
+    private Set<FlowApplication> applications;
```

Comment:
Java name and Json name are different. I think we should use same names (like we do everywhere)